templates/major.html

```diff
@@ -1,4 +1,36 @@
 {% extends "base.html" %}
+{% block head %}
```

Comment:
This will overwrite the `head` block from the base template, I meant to add the script tag to the head in the base template instead.

Reply:
I see. I didn't include it in `base.html` because `index.html`, which also extends `base.html`, doesn't need the `toggle` function (only `major.html` does).
And as `head` block is currently empty in `base.html`, I thought it acted as a placeholder.

Alternatively, we could call `super()` in `{% block head %}` ([reference documentation](https://jinja.palletsprojects.com/en/2.10.x/templates/#super-blocks)), so if content is added later on, it is not overwrote.

If you still think that it makes sense to have `toggle` on every page, I'd be happy to move the function directly to `base.html` though.

Reply:
Actually, I was replying from memory and was wrong! This is fine.